presto-oracle/src/test/java/io/prestosql/plugin/oracle/OracleServer.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import org.testcontainers.containers.OracleContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+
+public class OracleServer
+        implements Closeable
+{
+
+    public static final String TEST_SCHEMA = "test";
+    public static final String TEST_USER = "test";
+    public static final String TEST_PASS = "pwd4test";
+
+    private final OracleContainer oracle;
+
+    public OracleServer() throws SQLException {
+        oracle = new OracleContainer("wnameless/oracle-xe-11g-r2");
```

Comment:
Does `wnameless` hold a license allowing redistribution of the Oracle XE binaries?

Reply:
there was a long story of the `wnameless` distributive that got revoked from Docker Hub due to the DMCA notice (initiated by Oracle) but later they came to the agreement and the image re-appeared on Docker Hub.
Due to the way Docker Hub works, it wasn't possible to use the old name, hence `-r2`.